challenge-10/challenge-10.js

```diff
@@ -50,8 +69,19 @@ parâmetros;
 operador passado para a função "calculator", e passando para esse método
 os dois parâmetros da função de retorno de "calculator".
 */
-// ?
-
+	function calculator(operador) {
+		if( isOperatorValid(operador) === false ) {
+			return false;
+		} {
```

Comment:
-1
Aqui você está criando um bloco de código com o `{` sem necessidade.
Se a intenção era usar o `else`, faltou essa palavra entre os `}` e `{`.

Mas o ideal é que você não use `else`, já que você usa o `return` no `if`.
O `else` você só vai usar em casos muito raros, quando ele for realmente necessário. Evite sempre que possível.